modules/shared/src/main/scala/org/tessellation/security/Hashed.scala

```diff
@@ -8,3 +10,11 @@ import derevo.derive
 
 @derive(eqv, show)
 case class Hashed[A <: AnyRef](signed: Signed[A], hash: Hash, proofsHash: ProofsHash)
+
+object Hashed {
+  implicit def autoUnwrap[T <: AnyRef](t: Hashed[T]): T = t.signed
```

Comment:
FYI it uses `autoUnwrap` from `Signed` also to give the `T`. Not sure if it should use `Signed.autoUnwrap` or unwrap `Signed` here on its own.

Reply:
imo it's fine